src/kOS/KSPLogger.cs

```diff
@@ -23,7 +23,6 @@ public KSPLogger()
 
         public override void Log(string text)
         {
-            base.Log(text);
```

Comment:
This forced the logger to attempt to display the text on the screen as well as in the log. But the screen printing feature never got used, because the method was only called on the `SafeHouse` instance (which has no Shared object and therefore no Screen).  I did a "find all references" to be sure.  Since it isn't being used, I removed the reference to avoid confusion regarding the differing behavior.